<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Advanced_ctrl</class>
 <widget class="QDialog" name="Advanced_ctrl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>249</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string> 高级功能窗口</string>
  </property>
  <widget class="QLabel" name="label_Mac">
   <property name="geometry">
    <rect>
     <x>83</x>
     <y>13</y>
     <width>170</width>
     <height>19</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>24</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_buff">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>106</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_time">
   <property name="geometry">
    <rect>
     <x>83</x>
     <y>43</y>
     <width>170</width>
     <height>19</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>24</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_setbuff">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>106</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_transmode">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>165</y>
     <width>59</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1000</width>
     <height>1000</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <item>
    <property name="text">
     <string>TCP</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>EFAST</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_encrypt">
   <property name="geometry">
    <rect>
     <x>83</x>
     <y>73</y>
     <width>170</width>
     <height>19</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>24</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>165</y>
     <width>71</width>
     <height>19</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>24</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>传输模式 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Mac_2">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>13</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>24</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>MAC地址：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Mac_3">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>43</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>24</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>时        间：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Mac_4">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>73</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>24</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>公司信息：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Mac_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>106</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>24</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>模块视频缓存：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Mac_6">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>136</y>
     <width>100</width>
     <height>19</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>24</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>本地视频缓存：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_localvideobuff">
   <property name="geometry">
    <rect>
     <x>101</x>
     <y>136</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_setlocalvideobuff">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>136</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_synctime">
   <property name="geometry">
    <rect>
     <x>252</x>
     <y>42</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>同步时间</string>
   </property>
  </widget>
  <zorder>label_Mac</zorder>
  <zorder>lineEdit_buff</zorder>
  <zorder>label_time</zorder>
  <zorder>pushButton_setbuff</zorder>
  <zorder>comboBox_transmode</zorder>
  <zorder>label_encrypt</zorder>
  <zorder>pushButton_synctime</zorder>
  <zorder>label_7</zorder>
  <zorder>label_Mac_2</zorder>
  <zorder>label_Mac_3</zorder>
  <zorder>label_Mac_4</zorder>
  <zorder>label_Mac_5</zorder>
  <zorder>pushButton_synctime</zorder>
  <zorder>label_Mac_6</zorder>
  <zorder>lineEdit_localvideobuff</zorder>
  <zorder>pushButton_setlocalvideobuff</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
